migrator/README.md

```diff
@@ -18,17 +19,17 @@ A migration can read from any of the databases, make changes to the data or to t
 
    All migrations were of the form `m_{currentDBVersion}_to_m_{currentDBVersion+1}_{summary_of_migrations}` .
 
-2. Release 3.73 brought the database `Postgres` as Technical preview. This had the consequence that 
+2. Release 3.73 and 3.74 brought the database `Postgres` as Technical preview. This had the consequence that
 `Postgres` was a new potential datastore targeted by the migrator.
 
-   In 3.73, two sets of data migrations were possible: key-value store migrations, and data move migrations
+   In these versions, two sets of data migrations were possible: key-value store migrations, and data move migrations
    (from `BoltDB` and `RocksDB` to `Postgres`). The migration sequence is the following: first all key-value
    data migrations are applied, then all data move migrations are applied.
 
     - Key-value store data migrations have the form `m_{currentDBVersion}_to_m_{currentDBVersion+1}_{summary_of_migrations}` .
     - Data move migrations have the form `n_{postgresSchemaVersion}_to_n_{postgresSchemaVersion+1}_{moved_data_type}` .
     
-3. After 3.73, the key-value stores are deprecated and `Postgres` becomes the only data store.  
+3. After 4.0, the key-value stores are deprecated and `Postgres` becomes the only data store.
 
    The migration returns to the old scheme, restarting from the database version after all data moves to `Postgres`.
```

Comment:
This has been a source of confusion.  Might be worth clarifying and providing an example like after the last "data move migration" `n_56_to_n_57` we resume use of the normal migration pattern at `m_168_to_m_169`

Reply:
+1

Reply:
Em, it is at line 35.
 All migrations again have the form `m_{currentDBVersion}_to_m_{currentDBVersion+1}_{summary_of_migrations}`.

Reply:
It is, but I don't think it is clear especially with regard to the jump in numbers.  I think we should follow up line 35 with an example pointing out the gap.  I think it would help people understand.